opencsp/common/lib/csp/StandardPlotOutput.py

```diff
@@ -0,0 +1,556 @@
+            )
+            self.optic_measured.plot_orthorectified_slope_error(
+                self.optic_reference,
+                self.options_slope_vis.resolution,
+                type_='magnitude',
+                quiver_density=quiver_densities[0],
+                quiver_scale=quiver_scales[0],
+                quiver_color=quiver_colors[0],
+                clim=self.options_slope_vis.deviation_clim,
+                axis=fig_rec.axis,
+            )
+            if self._to_save:
+                fig_rec.save(
+                    output_dir=self.options_file_output.output_dir,
+                    dpi=self.options_file_output.save_dpi,
+                    format='png',
+                    close_after_save=self.options_file_output.close_after_save,
+                )
+
+            # Slope x
+            fig_rec = fm.setup_figure(
+                self.fig_control, self.axis_control, name="Slope Deviation X", number_in_name=False
+            )
+            self.optic_measured.plot_orthorectified_slope_error(
+                self.optic_reference,
+                self.options_slope_vis.resolution,
+                type_='x',
+                quiver_density=quiver_densities[1],
+                quiver_scale=quiver_scales[1],
+                quiver_color=quiver_colors[1],
+                clim=self.options_slope_vis.deviation_clim,
+                axis=fig_rec.axis,
+            )
+            if self._to_save:
+                fig_rec.save(
+                    output_dir=self.options_file_output.output_dir,
+                    dpi=self.options_file_output.save_dpi,
+                    format='png',
+                    close_after_save=self.options_file_output.close_after_save,
+                )
+
+            # Slope Y
+            fig_rec = fm.setup_figure(
+                self.fig_control, self.axis_control, name="Slope Deviation Y", number_in_name=False
+            )
+            self.optic_measured.plot_orthorectified_slope_error(
+                self.optic_reference,
+                self.options_slope_vis.resolution,
+                type_='y',
+                quiver_density=quiver_densities[2],
+                quiver_scale=quiver_scales[2],
+                quiver_color=quiver_colors[2],
+                clim=self.options_slope_vis.deviation_clim,
+                axis=fig_rec.axis,
+            )
+            if self._to_save:
+                fig_rec.save(
+                    output_dir=self.options_file_output.output_dir,
+                    dpi=self.options_file_output.save_dpi,
+                    format='png',
+                    close_after_save=self.options_file_output.close_after_save,
+                )
+        else:
+            lt.info('Do not have both measured and reference optic; skipping slope deviation plots.')
+
+    def _plot_ray_trace_image_measured_optic(self):
+        """Plot ray trace image for measured optic"""
```

Comment:
no need for doc-strings here. if you don't want this to potentially be pulled into the online documentation, I would convert to hash-tag comment.